readme.md

```diff
@@ -195,6 +195,24 @@ Default: `10000`
 
 Timeout in milliseconds for getting a response.
 
+##### onProgress
+
+Type: `Function`
+
+Progress event handler. The function takes `percent`, `transferred` and `total` arguments. If it's not possible to retrieve the body size, total will be `0`.
+
+
+```js
+await ky('https://example.com', {
+	onProgress: (percent, transferred, total) => {
```

Comment:
I think it might be nicer to make this an object instead. Could be called `progress`.

`transferred` => `transferredBytes`
`total` => `totalBytes`